iceoryx_posh/include/iceoryx_posh/roudi/memory/iceoryx_roudi_memory_manager.hpp

```diff
@@ -46,7 +46,6 @@ class IceOryxRouDiMemoryManager : public RouDiMemoryInterface
     /// MemoryBlocks to destroy their data
     cxx::expected<RouDiMemoryManagerError> destroyMemory() noexcept override;
 
-    PosixShmMemoryProvider* mgmtMemoryProvider() noexcept override;
     const PosixShmMemoryProvider* mgmtMemoryProvider() const noexcept override;
```

Comment:
Can you please update the copyright header so that it includes the years of the creation and of the last modification, in this case ``2020, 2021``?

Reply:
Same for the other changed files.

Reply:
I'm not sure if we need to update the date when we change an existing file. If yes, we should try to find a solution which doesn't require us to do it

Reply:
According to the ``CONTRIBUTING.md`` we have to update the year, but to be honest I would like to get rid of the year at all.

Reply:
@FerdinandSpitzschnueffler yes, it seems so. We need to discuss this in the meetup next week.

@JimmyBch sorry to bother you with this again, but the old date must also remain there. It must be `2020, 2021` or `2019, 2021`. Could you please change that. I promise we merge it then immediately :)

Reply:
Done :)